### PR TITLE
Flatten, de-dupe, and browse tags

### PR DESCRIPTION
**Pull request by rstorey**

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -65,3 +65,6 @@ flake8-assertive = "*"
 
 [requires]
 python_version = "3.6"
+
+[pipenv]
+allow_prereleases = true
```
Comment (acdha) [inline Pipfile]: This looks like a regression?

File: Pipfile [hunk 2]
```diff
@@ -13,7 +13,7 @@ django-haystack = "*"
 "boto3" = ">=1.9.16"
 botocore = '>=1.12.16'
 requests = "*"
-Django = ">=2.1.7"
+Django = "===2.1.7"
```
Comment (acdha) [inline Pipfile]: Should this be `~=2.1.8`?

Comment (rstorey) [reply Pipfile]: I'm resetting the pipfile for this branch... there's no reason for it to be different from master

Comment (acdha) [reply Pipfile]: Ah, good

**Comment by rstorey** [comment]
Closing until tagging becomes a priority

**Pull request closed without merging**